packages/browser-extension/source/DevToolScript/index.tsx

```diff
@@ -109,6 +123,52 @@ browser.devtools.panels
       //panelWindow.respond = function (msg) {
       //  port.postMessage(msg);
       //};
+
+      console.log("sk_SK stringify",JSON.stringify(sk_SK))
+      
+      const loadedFile = await loadFileFromReactProject('/Users/michalzaduban/Desktop/talentsbase/src/localizations/en_EN.json')
+      //@ts-ignore
```

Comment:
why do we have `//@ts-ignore` here?

Reply:
bc loadedFile is string | void and function createAst accepts string as the first argument